docs/tutorials/pasqal/getting_started.ipynb

```diff
@@ -48,16 +48,16 @@
    "source": [
     "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
     "  <td>\n",
-    "    <a target=\"_blank\" href=\"https://www.example.org/cirq/pasqal/getting_started\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on QuantumLib</a>\n",
+    "    <a target=\"_blank\" href=\"https://www.example.org/cirq/pasqal/getting_started\"><img src=\"https://www.tensorflow.o../../images/tf_logo_32px.png\" />View on QuantumLib</a>\n",
```

Comment:
Images are broken by this change.

Reply:
Ahh great catch - it was a replace all error.